### ISSUE TITLE
Include MEGAHIT assembly

### ISSUE THREAD

**Issue by abremges** [metagenomics/2015-biogas-cebitec#31]
The dataset should also be assembled with MEGAHIT (appeared in 2015 in Bioinformatics) for comparison. Ray Meta is 3 years old.

**Comment by abremges** [comment]
Related to #14

**Comment by abremges** [comment]
I just started the MEGAHIT assembly to compare. Won't switch to another assembly, though, as groups are working on the Ray one (which is more than decent already!). Still interested in the results...

**Comment by abremges** [comment]
Assembly | Total bases | No. Contigs | N50 | "NG50_Ray" | "NG50_Mega" | Largest Contig
--- | --- | --- | --- | --- | --- | ---
Ray_k31 | 228,382,457 | 54,489 | 9,796 | 9,796 | 3,400 | 333,979
MEGAHIT | 319,169,696 | 113,513 | 3,566 | 5,587 | 3,566 | 197,663

"NG50" is a pseudo-normalized N50 value: I took the total assembly size of *Ray_k31* or *Megahit* as the genome length. This way, we can compare the contiguity between both assemblies.

So in conclusion, judging by this crude measure we assemble more of the metagenome with Metahit, but the "upper part" of the assembly is more fragmented. Will run a gene prediction next.

**Comment by asczyrba** [comment]
Interesting. How many reads map back to these assemblies?

**Comment by abremges** [comment]
For the Ray_k31 assembly, see #13
I did not map the reads in the new assemblies, yet.

**Comment by abremges** [comment]
Mapping rates are in the 8x range for the MEGAHIT assembly. It might be better at reconstructing low-abundance players, and thus does not include that many more reads (but it's only a first and uneducated guess). Alternative explanations are a higher duplication rate, but I haven't looked into it.

As there are people working on our Ray_k31 assembly already, and we have no extensive experience with MEGAHIT's assembly quality (but know Ray quite well and in general are very happy with its results), I will argue like this in our response letter - closing this.
